### DISCUSSION TITLE
unable to read resource file

### DISCUSSION THREAD

**Discussion by Beyond-Koma** [dnSpyEx/dnSpy#162] [Q&A]
![1](https://user-images.githubusercontent.com/12950538/214705039-c88101a5-821d-4954-9e54-c190229bbe5e.png)

**Comment by ElektroKill** [comment]
Hi, the file you are currently trying to open in dnSpy is a .NET 6 single file bundle file. This is a different file format then the regular .NET PE images dnSpy can open. It is possible to add support for this format (as you can see ILSpy has done so) and it is on my list of things to add to dnSpy. The only way as of right now to analyze such files in dnSpy is to extract them with a different utility first and then open them in dnSpy.

**Reply by Beyond-Koma** [reply]
Thank you for your reply.Hope to support later.
